expat/configure.ac

```diff
@@ -43,8 +43,8 @@ dnl   DAMAGES OR  OTHER LIABILITY, WHETHER  IN AN  ACTION OF CONTRACT,  TORT OR
 dnl   OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE
 dnl   USE OR OTHER DEALINGS IN THE SOFTWARE.
 
-dnl Ensure that Expat is configured with autoconf 2.69 or newer.
-AC_PREREQ([2.69])
+dnl Ensure that Expat is configured with autoconf 2.71 or newer.
+AC_PREREQ([2.71])
```

Comment:
This bump in requirement is not needed and may lock some users out, given that 2.71 was released only 2021.01, please revert this part.

Reply:
PS: Turned out to be more of a CI than a user problem if assuming that users consume `make dist` output where `configure` has already been rendered and Autoconf does not need to be installed. In CI however, Ubuntu 20.04 does not have Autoconf >=2.70 available, the minimum needed to drop use of macro `AC_PROG_CC_C99`, see https://www.gnu.org/software/autoconf/manual/autoconf-2.70/html_node/Obsolete-Macros.html#index-AC_005fPROG_005fCC_005fC99-1 . There was no way to bump to 2.71 here.